### PR TITLE
Emo loading screens

### PR DESCRIPTION
**Pull request by siddharthkp**
Goes well with https://github.com/codesandbox/codesandbox-client/pull/2941

- [x] common/RunOnClick
- [x] sandbox/loading-screen
- [x] sandbox/run-on-click

## Checklist

- [NA] Documentation
- [x] Testing
- [ ] Ready to be merged

I'd really like if someone can test this @CompuIves

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/2993/commits/ada26e3b2cbf7f625a978feb450d2b58ef638908 is at https://pr2993.build.csb.dev/s/new.

**Comment by SaraVieira** [comment]
https://www.youtube.com/watch?v=dhZTNgAs4Fc

**Comment by siddharthkp** [comment]
Screenshot 2019-11-07 at 8 46 32 AM

> It just takes some time,
> little girl your in the middle of the ride,
> everything, everything will be just fine.
https://open.spotify.com/album/0UJhhj5bn5AGAjryFnhueP

File: packages/app/src/sandbox/status-screen/loading-screen.html [hunk 1]
```diff
@@ -1,516 +1,240 @@
...
-    50% {
-      -webkit-transform: rotateX(-37.5deg) rotateY(405deg);
-      transform: rotateX(-37.5deg) rotateY(405deg);
-    }
-    100% {
-      -webkit-transform: rotateX(-37.5deg) rotateY(405deg);
-      transform: rotateX(-37.5deg) rotateY(405deg);
-    }
+  .text {
+    margin-top: 24rem;
```
Comment (CompuIves) [inline packages/app/src/sandbox/status-screen/loading-screen.html]: Small change!

```suggestion
    margin-top: 22rem;
```

**Pull request merged**